build/nlu-installer/src/installer.ts

```diff
@@ -25,6 +27,10 @@ const makeFileNamePerOS = (version: string): Partial<PerOS<string>> => {
   }
 }
 
+const removeConsecutiveSlashes = (url: string) => {
+  return url.replace(/([^:]\/)\/+/g, '$1')
+}
+
```

Comment:
No change needed there, but I'm just suggesting quick return for future stuff `const rem = (url) => url.replace()`